src/database/sqlite/event/event_database.py

```diff
@@ -1854,14 +1853,18 @@ def _row_to_stored_board(cls, row: dict[str, Any]) -> StoredBoard:
             white_player_id=row['white_player_id'],
             black_player_id=row['black_player_id'],
             index=row['index'],
+            last_result_update=row.get('last_result_update'),
         )
 
     def load_tournament_stored_boards_by_round(
         self, tournament_id: int
     ) -> dict[int, list[StoredBoard]]:
         self.execute(
             (
-                'SELECT `board`.*, `pairing`.`round` '
+                'SELECT `board`.`id`, `board`.`white_player_id`, `board`.`black_player_id`, '
+                '`board`.`index`, '
+                'datetime(`board`.`last_result_update`, "unixepoch") as last_result_update, '
+                '`pairing`.`round` '
```

Comment:
```suggestion
                'SELECT `board`.*, `pairing`.`round` '
```
It's important to maintain the `.*` select to avoid maintenance issues. I'm all for putting datetimes in the DB, but if that means having all the `SELECT *` be replaced by the list of column it's not worth it (would be a nightmare to maintain for events or tournaments)